odkx-src/sync-endpoint-manual-setup.rst

```diff
@@ -98,7 +98,9 @@ Setup instructions:
 
     .. note::
 
-      It is important that the right side of the colon stays as 8080. This is the internal port that the web server is looking for.
+      It is important that the right side of the colon stays as 8080 or whatever port you are using (8080 is the default). This is the port that the web server is looking for.
```

Comment:
This part does not seem to be about the QR scanner, does it relate to another open pull request?

Reply:
I'll just delete this part